src/main/java/com/jcabi/aether/Aether.java

```diff
@@ -91,10 +91,6 @@
  * @checkstyle ClassFanOutComplexity (500 lines)
  * @see <a href="http://sonatype.github.com/sonatype-aether/apidocs/overview-tree.html">Aether 1.13.1 JavaDoc</a>
  * @see Classpath
- * @todo #11:30min Create an integration test for reading proxy information
- *  from maven settings.xml file that will mock an HTTP server
- *  using jcabi-http, provide a maven configuration file with proxy settings
- *  and verify the expected HTTP requests have been sent to the mocked server.
```

Comment:
@caarlos0 Could you please let me know which root pom you are referring to. Because I could not find a place where similar IT tests were added. (mirror-usage and basic-usage).